src/components/SupportRequestModal.js

```diff
@@ -51,7 +55,7 @@ const SupportRequestModal = ajaxCaller(class SupportRequestModal extends Compone
         isMulti: false,
         value: type,
         onChange: ({ value }) => this.setState({ type: value }),
-        options: [{ value: 'bug', label: 'Bug' }, { value: 'question', label: 'Question' }, { value: 'feature_request', label: 'Feature Request' }]
+        options: [{ value: 'question', label: 'Question' }, { value: 'bug', label: 'Bug' }, { value: 'feature_request', label: 'Feature Request' }]
```

Comment:
Again, is there a reason you changed this?

Reply:
(see above...)